src/java.base/share/classes/sun/nio/ch/IOUtil.java

```diff
@@ -475,15 +475,15 @@ static long read(FileDescriptor fd, ByteBuffer[] bufs, int offset, int length,
 
     private static final JavaNioAccess NIO_ACCESS = SharedSecrets.getJavaNioAccess();
 
-    static Scope.Handle acquireScope(ByteBuffer bb, boolean async) {
+    static Runnable acquireScope(ByteBuffer bb, boolean async) {
```

Comment:
At some point (not this PR) we should move the "async" out of this file, IOUtil was for synchronous I/O.